packaging/linux/Dockerfile

```diff
@@ -28,7 +28,14 @@ RUN apt-get install -y nodejs yarn
 
 # Install Go directly from Google, because the Debian repos tend to be behind.
 # When updating the version, remember to bump keybase_packaging_v# in docker_build.sh.
-RUN wget "https://storage.googleapis.com/golang/go1.12.4.linux-amd64.tar.gz" -O /root/go.tar
-RUN tar -C /usr/local -xzf /root/go.tar
-RUN rm /root/go.tar
+# Copy over the new hash when upgrading version. But if not upgrading version,
+# the check should not ever fail.
+ENV GOLANG_VERSION 1.12.4
```

Comment:
should we bump to 1.12.5 if we need that for mobile anyway?

Reply:
Thought we needed something not released yet for mobile?